downstream/modules/platform/proc-controller-deleting-a-user.adoc

```diff
@@ -0,0 +1,16 @@
+[id="proc-controller-deleting-a-user"]
+
+= Deleting a user
+
+Before you can delete a user, you must have user permissions. 
+When you delete a user account, the name and email of the user are permanently removed from {ControllerName}.
+
+.Procedure
+. In the navigation panel, expand the *Access*.
```

Comment:
The steps in the guide are currently inconsistent, maybe it is something I can correct in the whole guide before we publish. After editorial review and confirming accessibility options with Donna, I re-wrote these as:

. From the navigation panel, select menu:Access[Users].